d3a_api_client/aggregator.py

```diff
@@ -107,16 +108,11 @@ def _all_uuids_in_selected_device_uuid_list(self, uuid_list):
                 raise Exception(f"{device_uuid} not in list of selected device uuids")
         return True
 
-    def batch_command(self, batch_command_dict):
-        """
-        batch_dict : dict where keys are device_uuids and values list of commands
-        e.g.: batch_dict = {
-                        "dev_uuid1": [{"energy": 10, "rate": 30, "type": "offer"}, {"energy": 9, "rate": 12, "type": "bid"}],
-                        "dev_uuid2": [{"energy": 20, "rate": 60, "type": "bid"}, {"type": "list_market_stats"}]
-                        }
-        """
+    def batch_command(self):
```

Comment:
I think I should rename this method to something more meaningful

Reply:
Maybe `execute_batch_commands` or `send_batch_commands`